CMakeLists.txt

```diff
@@ -57,6 +57,17 @@ endif(CMAKE_COMPILER_IS_GNUCXX)
 option(BUILD_TESTS "Configure CMake to build tests" ON)
 option(BUILD_BENCHMARKS "Configure CMake to build (google) benchmarks" OFF)
 
+###################################################################################################
+# - cudart options --------------------------------------------------------------------------------
+# cudart can be statically linked or dynamically linked the python ecosystem wants dynamic linking
+
+
+if(CMAKE_CUDA_RUNTIME_LIBRARY MATCHES "^Static$")
```

Comment:
Sorry this does not actually work to default to dynamic on newer versions of CMake and CMake itself is setting it to Static by default.  We could switch to use a different config or I could look int seeing if it was set on the command line some how.